packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -135,6 +134,28 @@ class MuxUploaderElement extends globalThis.HTMLElement implements MuxUploaderEl
     this._endpoint = value;
   }
 
+  get noDrop(): boolean {
+    return this.hasAttribute('noDrop');
+  }
+
+  set noDrop(value: boolean) {
+    if (value) {
+      this.setAttribute('noDrop', '');
+    } else {
+      this.removeAttribute('noDrop');
+    }
+    this.updateLayout();
```

Comment:
this should be called on in the `attributeChangedCallback` I think if we like attribute changes to have effect on the layout as well. not only setting the `noDrop` prop. requires `nodrop` to be added to the `static observedAttributes: []` as well